### PR TITLE
Hello World example for Automic 

### PR DESCRIPTION
**Pull request by indraneel07**

### CODE REVIEW

File: orbiter_translations/automic/xml_demo.py [hunk 1]
```diff
@@ -0,0 +1,359 @@
...
+    ```
+    """  # noqa: E501
+    task_definitions = {}
+    tasks = []
+    for dag_struct_key, struct in val.items():
+        if dag_struct_key in ["SCRI", "JOBS"]:
+            for task_details in struct:
+                task_definitions[task_details["@name"]] = task_details
+
+    for dag_struct_key, struct in val.items():
```
Comment (fritz-astronomer) [inline orbiter_translations/automic/xml_demo.py:223]:
```suggestion
```

File: orbiter_translations/automic/xml_demo.py [hunk 2]
```diff
@@ -0,0 +1,359 @@
...
+                return OrbiterSQLExecuteQueryOperator(
+                    task_id=val["@OH_TITLE"],
+                    sql=val["script"]["SCRIPT"].split(":SQL")[-1].strip(),
+                    **conn_id(conn_id="mssql_default", conn_type="mssql"),
+                )
+        return OrbiterEmptyOperator(task_id=val["@OH_TITLE"])
+
+    except StopIteration:
+        pass
+    return None
```
Comment (fritz-astronomer) [inline orbiter_translations/automic/xml_demo.py:273]: can you split these into three separate rules? One returning an `OrbiterEmptyOperator` (or modify the `cannot_map_rule`), one returning a `OrbiterSQLExecuteQueryOperator`, and one returning a `OrbiterBashOperator` ?

**Review by fritz-astronomer — CHANGES_REQUESTED** [review]
Excellently done! Can you split that `@task_rule` into three `@task_rules`? And then good to merge!

File: orbiter_translations/automic/xml_demo.py (hunk 2 above)
Comment (indraneel07) [reply orbiter_translations/automic/xml_demo.py]: Have split the task rules, as suggested.

**Review by fritz-astronomer — APPROVED** [review]

**Pull request merged**